packages/annotations/src/search.ts

```diff
@@ -105,3 +108,160 @@ export function searchAnnotations(
     });
   });
 }
+
+export interface IVoteResourceObject {
+  id: number;
+  type: "votes";
+  attributes: {
+    upVotes: number;
+    downVotes: number;
+  };
+}
+
+/**
+ * ```js
+ * import { searchAnnotationVotes } from "@esri/hub-annotations";
+ * //
+ * searchAnnotationVotes({ url: annotationsUrl + "/0"},
+ *                         annotation: {
+ *                            id: "Annotation1",
+ *                            type: "annotations",
+ *                            attributes: {description: "Great place!", ...}
+ *                          })
+ *   .then(response => {
+ *     //   data: [{
+ *     //     id,
+ *     //     type: "votes",
+ *     //     attributes: {
+ *     //       upVotes: 3,
+ *     //       downVotes: 0
+ *     //     }
+ *     //   }]
+ *    });
+ * ```
+ * Query for up and down votes on a comment from ArcGIS Hub.
+ * @param requestOptions - request options that may include authentication
+ * @param annotation - the annotation for which votes need to be counted
+ * @returns A Promise that will resolve with summary statistics for the specified annotation from the annotation service for a Hub enabled ArcGIS Online organization.
+ */
+export function searchAnnotationVotes(
+  requestOptions: IQueryFeaturesRequestOptions,
+  annotation: IAnnoFeature
+): Promise<{ data: IVoteResourceObject[] }> {
+  requestOptions.groupByFieldsForStatistics = "value";
+  const outStat: IStatisticDefinition = {
+    statisticType: "count",
+    onStatisticField: "value",
+    outStatisticFieldName: "value_count"
+  };
+  requestOptions.outStatistics = [outStat];
+  // filtering for the comment
+  const commentFilteringClause = "parent_id=" + annotation.attributes.OBJECTID;
+  requestOptions.where += requestOptions.where ? "+AND+" : "";
```

Comment:
is this `where` check unnecessary?

Reply:
my gut reaction is `yes'.

i don't think this method needs to support blending a filter provided in `IQueryFeaturesRequestOptions` when querying for summary stats on the annotation it already has a reference to.